src/Imgix/UrlBuilder.php

```diff
@@ -2,8 +2,6 @@
 
 namespace Imgix;
 
-use Imgix\Validator;
```

Comment:
Could you elaborate on why we can get away with removing this line?

Reply:
Since all classes are defined in the same namespace (`\Imgix`), they can always reach/see each other.

If the validator class had been defined in another namespace, we would have to define the import statement, for example `\Imgix\Validators\Validator`.

You can think of the namespaces as folders, and if the files are in the same folder they can see each other without needing an import statement.

```php
namespace A;

use B\D;
use C\E as F;

// ...

// class references

new B();    // creates object of class "B" defined in namespace "A"
            // if not found, it tries to autoload class "A\B"

new D();    // using import rules, creates object of class "D" defined in namespace "B"
            // if not found, it tries to autoload class "B\D"

new F();    // using import rules, creates object of class "E" defined in namespace "C"
            // if not found, it tries to autoload class "C\E"

new \B();   // creates object of class "B" defined in global scope
            // if not found, it tries to autoload class "B"

new \D();   // creates object of class "D" defined in global scope
            // if not found, it tries to autoload class "D"

new \F();   // creates object of class "F" defined in global scope
            // if not found, it tries to autoload class "F"

// ...
```

Source: [PHP Docs](https://www.php.net/manual/en/language.namespaces.rules.php)

As you can see above, the class `B` is not imported at the top, and will be assumed to be in the *current* namespace, `A`.

Please let me know if I need to clarify anything! 👍